### ISSUE TITLE
Test failures with Sinatra 2.1.0+ / Ruby 3.0+

### ISSUE THREAD

**Issue by voxik** [sinatra/sinatra#1775]
Working on #1774 and looking at the https://github.com/seattlerb/minitest/issues/894 referred in #1772, I observe one more issue of this kind:

~~~diff
  1) Failure:
HAMLTest#test_renders_with_inline_layouts_0 [/builddir/build/BUILD/sinatra-2.2.0/usr/share/gems/gems/sinatra-2.2.0/test/haml_test.rb:34]:
--- expected
+++ actual
@@ -1,2 +1,5 @@
-"<h1>THIS. IS. <EM>SPARTA</EM></h1>
+# encoding: ASCII-8BIT
+#    valid: true
+"<h1>THIS. IS. <EM>SPARTA</EM>
+</h1>
 "


  2) Failure:
HAMLTest#test_renders_with_file_layouts_0 [/builddir/build/BUILD/sinatra-2.2.0/usr/share/gems/gems/sinatra-2.2.0/test/haml_test.rb:40]:
--- expected
+++ actual
@@ -1,3 +1,6 @@
+# encoding: ASCII-8BIT
+#    valid: true
 "<h1>HAML Layout!</h1>
-<p>Hello World</p>
+<p>Hello World
+</p>
 "
~~~

I think that this time, it might be caused by did_you_mean. However, disabling it does not help.

**Comment by dentarg** [comment]
Does those tests fail on master too?

**Comment by dentarg** [comment]
@voxik still an issue?

**Comment by dentarg** [comment]
@jkowens I think we can close this issue now. @voxik can create a new issue if it is still an issue with the latest release.

**Comment by voxik** [comment]
Just to close the loop, I believe this was fixed by #1698
